### PR TITLE
Add Task #3 Test Code

### PR DESCRIPTION
**Pull request by vendettadelsangue**
# ForceHttps Test
> ## When OpenApi__ForceHttps is true, Server's url should be return https.

### CODE REVIEW

File: test-integration/Microsoft.Azure.WebJobs.Extensions.OpenApi.Document.Tests/Configurations/OpenApiConfigurationOptionsTests.cs [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            license.Name.Should().Be(OpenApiInfoConfigs.LicenseName);
+        }
+
+        [TestMethod]
+        public void Given_OpenApiDocument_Then_It_Should_Return_LicenseUrl()
+        {
+            var license = this._doc.Info.License;
+
+            license.Url.AbsoluteUri.Should().Be(OpenApiInfoConfigs.LicenseUrl);
+        }
```
Comment (justinyoo) [inline test-integration/Microsoft.Azure.WebJobs.Extensions.OpenApi.Document.Tests/Configurations/OpenApiConfigurationOptionsTests.cs:94]: These tests are not relevant to your issue. Would you please remove it?

**Review by justinyoo — CHANGES_REQUESTED** [review]
@vendettadelsangue Thanks for your PR! I left some comments on it. Please take a look and update your PR.

File: test-integration/Microsoft.Azure.WebJobs.Extensions.OpenApi.Document.Tests/Configurations/OpenApiConfigurationOptionsTests.cs (hunk 1 above)
Comment (vendettadelsangue) [reply test-integration/Microsoft.Azure.WebJobs.Extensions.OpenApi.Document.Tests/Configurations/OpenApiConfigurationOptionsTests.cs]: Thanks for the code review.
I deleted the code and committed it again.

**Review by justinyoo — APPROVED** [review]
LGTM! Thanks for your contribution!

**Pull request merged**